src/components/DifficultyExplanation/DifficultyExplanation.css

```diff
@@ -0,0 +1,20 @@
+.information{
```

Comment:
i noticed that on slimmer screen sizes, the difficulty explanation text spills outside of the information box. you could try fixing that in this PR or put it off to a separate issue, your choice!